pkg/cli/run.go

```diff
@@ -326,6 +327,10 @@ func (s *Run) update(ctx context.Context, c client.Client, imageSource imagesour
 		return nil, false, err
 	}
 
+	if s.Region != "" && app.GetRegion() != s.Region {
+		pterm.Warning.Println("The region of an app can not be changed, ignoring --region")
```

Comment:
> Maybe it's worth making this consistent? What are your thoughts?

If we continue here, how far downstream does the operation propagate before failing in the same way `kubectl edit` does? does it fail at all?

Reply:
The logic would have to be altered, but it would fail at the API request.

Right now, the region isn't propagated to the API request. It is ignored.